src/main/java/com/sampleweb/pages/BooksPage.java

```diff
@@ -0,0 +1,68 @@
+package com.sampleweb.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.util.List;
+
+public class BooksPage extends HomePage {
+
+    public String title = "Books";
+    public String pageTitle = "//*[@id=\"merchandised-content\"]/div[1]/div[1]/div/h1";
+    public String globalStore = "//*[@id=\"leftNav\"]/h4[5]";
+
+    public static String PATH = "https://www.amazon.co.uk";
+
+    public BooksPage(RemoteWebDriver driver) {
+        super(driver);
+    }
+
+    public void selectBooks(){
+        //Select dropdown menu
+        driver.findElement(By.xpath("//*[@id=\"searchDropdownBox\"]")).click();
+        //Select option books
+        driver.findElement(By.xpath("//*[@id=\"searchDropdownBox\"]/option[10]")).click();
+        //Submit search
+        driver.findElement(By.xpath("//*[@id=\"nav-search\"]/form/div[2]/div/input")).click();
```

Comment:
keep a consistent approach of storing these in appropriately named variables so you can know what element it is referring to